src/messages/layout.js

```diff
@@ -47,6 +47,19 @@ export const TYPE_NO_MANIFEST_JSON = {
     will be attempted to be inferred by package layout.`),
 };
 
+export const TYPE_MULTIPLE_FILES = {
```

Comment:
I think this might be better as `MULTIPLE_MANIFEST_FILES` or `MULTIPLE_MANIFESTS`